src/collectd.conf.pod

```diff
@@ -5438,6 +5438,17 @@ behavior is to let the kernel choose the appropriate interface. Be warned
 that the manual selection of an interface for unicast traffic is only
 necessary in rare cases.
 
+=item B<BindAddress> I<IP Address>
+
+Set the outgoing IP address for IP packets. This option can be used instead of
+the I<Interface> option to explicitly define the IP address which will be used
+to send Packets to the remote server. This applies at least
+to IPv6 packets and if possible to IPv4. If this option is not applicable,
+undefined or a non-existent interface name is specified, the default
```

Comment:
I think that's too vague. What does it mean to not be applicable? I suggest something like 'if this option is not set'

Reply:
This is copy/paste from the Interface Option and it acts exactly the same.
it's either the Interface option or BindAddress option or none. Since this is the description in the Interface option I left it as is because of the same functionality.

Reply:
You are correct about the Interface rather than the IP but since the default settings will be the interface as it was before the change the documentation as is, is correct. The default is binding to an interface selected by the kernel as it was before.

Reply:
You are correct about the Interface rather than the IP but since the default settings will be the interface as it was before the change the documentation as is, is correct. The default is binding to an interface selected by the kernel as it was before.

Reply:
>The default is binding to an interface selected by the kernel as it was before.

That is not true, IMHO. There is no _binding_ by default, so if you change routing after Collectd start, packets will flow by new routing rules.

https://linux.die.net/man/7/socket : 

>If the name is an empty string or the option length is zero, the socket device binding is removed.

So, there is no binding by default, as I understand.

Reply:
Ok. I will drop the lines below the third sentence